media-source-respec.html

```diff
@@ -1160,15 +1223,67 @@ <h2><dfn>MediaSourceHandle</dfn> Object</h2>
       <p class="note">This distinct object is necessary to attach a cross-context {{MediaSource}} to a media element
         because {{MediaSource}} objects themselves are not transferable since they are event targets.</p>
 
+      <p>Each {{MediaSourceHandle}} object has a <dfn data-dfn-for="MediaSourceHandle">[[\has ever been assigned as
+        srcobject]]</dfn> internal slot that stores a {{boolean}}. It is initialized to false when the
+        {{MediaSourceHandle}} object is created, is set true in the extended {{HTMLMediaElement}}'s
+        {{HTMLMediaElement/srcObject}} setter as described in
+        <a href="#htmlmediaelement-extensions">HTMLMediaElement Extensions</a>, and if true, prevents successful
+        transfer of the {{MediaSourceHandle}} as described in the [=Transfer=] section.</p>
+
+      <p>{{MediaSourceHandle}} objects are {{Transferable}}, each having a <dfn
+        data-dfn-for="MediaSourceHandle">[[\Detached]]</dfn> internal slot that is used to ensure that once the handle
+        object has been transferred, it cannot be transferred again.</p>
```

Comment:
Is it true it cannot be transferred multiple times before being used ?

Reply:
Once one handle has been transferred, the new handle can be transferred again.

Reply:
The internal slot is not part of the transfer. It is set true on the handle instance that is transferred. The receiver of the transfer has a new handle instance with this [[Detached]] internal slot initialized to false. This is standard behavior of the transfer steps. For minor improved clarity, I'll change "object" to "object instance", but I suspect those familiar with web platform transfer semantics will understand without that change. If I'm mistaken, we can further clarify in later PR.

Reply:
Note, I took this language directly from the [Transferable](https://html.spec.whatwg.org/multipage/structured-data.html#transferable) spec:

> [Platform objects](https://webidl.spec.whatwg.org/#dfn-platform-object) that are [transferable objects](https://html.spec.whatwg.org/multipage/structured-data.html#transferable-objects) have a [[Detached]] internal slot. This is used to ensure that once a platform object has been transferred, it cannot be transferred again.

Reply:
Done in f247a63